build.gradle

```diff
@@ -111,6 +111,8 @@ dependencies {
     testImplementation 'org.codehaus.groovy.modules.http-builder:http-builder:0.7.1'
     testImplementation 'org.springframework.boot:spring-boot-starter-test'
     testImplementation 'junit:junit:4.13'
+    testImplementation(platform("org.junit:junit-bom:5.6.0"))
+    testImplementation("org.junit.jupiter:junit-jupiter")
```

Comment:
I think this will be included already by spring boot test so it could be removed. Not sure what `bom` do you need it?

Reply:
Removed.